src/classes/RD_RecurringDonations_TDTM.cls

```diff
@@ -71,6 +72,8 @@ public class RD_RecurringDonations_TDTM extends TDTM_Runnable {
             newlist = oldlist;
         }
 
+        autogenerateNames(newlist, oldlist, triggerAction);
```

Comment:
@force2b , the current version will have automated naming applied. When the new RD version is enabled, the enablement and RD2 infrastructure will be developed in W-034362, so we should ensure naming is applied there. Let me know if there is any objection.